gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.0.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
```

Comment:
Is there actually requirement that the gradle versions must be the same? I mostly wanted to upgrade because Gradle got much faster in 4.5. I can take AI to upgrade the rest if you'd like?

Reply:
+1 to @pongad - there is no inherent reason these need to be consistent.

Reply:
I agree that it is not required, but seems nice to have, especially if gradle 4.5 is significantly better.